tee/prototype/epochproc.go

```diff
@@ -101,7 +101,7 @@ func (e *Enclave) txRoutine(
 
 			bps, err := e.generateBalanceProofs(txEpoch)
 			if err != nil {
-				return fmt.Errorf("generating balance proofs: %w", err)
+				log.Errorf("generating balance proofs: %w", err)
```

Comment:
if this errors, it's really broken, so better leave as is. (note that you shouldn't use `%w` in logging)